firebase-storage/src/main/java/com/google/firebase/storage/StorageRegistrar.java

```diff
@@ -17,35 +17,41 @@
 import androidx.annotation.Keep;
 import androidx.annotation.RestrictTo;
 import com.google.firebase.FirebaseApp;
+import com.google.firebase.annotations.concurrent.Blocking;
 import com.google.firebase.appcheck.interop.InternalAppCheckTokenProvider;
 import com.google.firebase.auth.internal.InternalAuthProvider;
 import com.google.firebase.components.Component;
 import com.google.firebase.components.ComponentRegistrar;
 import com.google.firebase.components.Dependency;
+import com.google.firebase.components.Qualified;
 import com.google.firebase.platforminfo.LibraryVersionComponent;
 import java.util.Arrays;
 import java.util.List;
+import java.util.concurrent.Executor;
 
 /** @hide */
 @Keep
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 public class StorageRegistrar implements ComponentRegistrar {
   private static final String LIBRARY_NAME = "fire-gcs";
+  Qualified<Executor> backgroundExecutor = Qualified.qualified(Blocking.class, Executor.class);
```

Comment:
It's better to be consistent with naming, rename this to `blockingExecutor`

Reply:
Nice catch. Fixed